openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -52,8 +52,8 @@ def get_account_settings(request, username=None, configuration=None, view=None):
         request (Request): The request object with account information about the requesting user.
             Only the user with username `username` or users with "is_staff" privileges can get full
             account information. Other users will get the account fields that the user has elected to share.
-        username (str): Optional username for the desired account information. If not specified,
-            `request.user.username` is assumed.
+        username (str): Optional string of comma separated usernames for the desired account information. If not
+            specified, `request.user.username` is assumed.
```

Comment:
The view should probably process the comma separated usernames, so the api function can take a list of usernames.